sdk/core/azure-core-http-okhttp/CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Release History
+
+## 1.2.0-beta.1 (Unreleased)
+
+## Version 1.1.0 (2019-11-26)
+- Switch to JUnit version 5.
```

Comment:
I do not see okhttp 1.1.0  released yet http://repo2.maven.apache.org/maven2/com/azure/azure-core-http-okhttp/